lib/sheets/index.ts

```diff
@@ -50,6 +51,12 @@ class Sheets {
     results.forEach(data => {
       const getTiming = (key: string) => data.timings.find(t => t.id === key).timing;
       const dateObj = new Date(data.generatedTime);
+      let viewerUrl = '';
+
+      if(typeof data.fileName !== 'undefined' && typeof data.fileName !== 'undefined') {
```

Comment:
Huh - just noticed one of those is supposed to be data.fileId, but that's the only one needed now - I originally intended to use both, but ditched the idea of using the file name as the link text.

I use `typeof` rather than a truthy check as it's won't choke on undefined.